implementation/service_discovery/src/service_discovery_impl.cpp

```diff
@@ -869,7 +871,7 @@ service_discovery_impl::create_eventgroup_entry(
                 << std::setw(4) << _service << "."
                 << std::setw(4) << _instance << "."
                 << std::setw(4) << _eventgroup << "] "
-                << (uint16_t) _reliability_type;
+                << +static_cast<uint16_t>(_reliability_type);
```

Comment:
Remove typo.

```suggestion
                << static_cast<uint16_t>(_reliability_type);
```

Reply:
Not a typo, see https://github.com/COVESA/vsomeip/pull/560#discussion_r1550589301